src/ui/public/paginated_table/paginated_table.js

```diff
@@ -46,42 +44,49 @@ uiModules
 
         self.sort.columnIndex = colIndex;
         self.sort.direction = sortDirection;
-        self._setSortGetter(colIndex);
+        if ($scope.sort) {
+          _.assign($scope.sort, self.sort);
+        }
       };
 
-      self._setSortGetter = function (index) {
-        if (_.isFunction($scope.sortHandler)) {
-          // use custom sort handler
-          self.sort.getter = $scope.sortHandler(index);
-        } else {
-          // use generic sort handler
-          self.sort.getter = function (row) {
-            var value = row[index];
-            if (value && value.value != null) value = value.value;
-            if (typeof value === 'boolean') value = value ? 0 : 1;
-            return value;
-          };
+      function valueGetter(row) {
+        let value = row[self.sort.columnIndex];
+        if (value && value.value != null) value = value.value;
```

Comment:
Should this be `!==` or are you intentionally checking for null-like instead of null?